statistics/table.go

```diff
@@ -42,41 +42,53 @@ const (
 
 // Table represents statistics for a table.
 type Table struct {
+	*HistColl
```

Comment:
Can this be nil?
If not, we can use struct instead of pointer.

Reply:
Okay. got it